interactor-impl/src/test/java/com/wednesday/template/interactor/lastFm/search/SearchAlbumInteractorImplTest.kt

```diff
@@ -62,7 +75,7 @@ class SearchAlbumInteractorImplTest : InteractorTest() {
                     val result = awaitItem()
 
                     // Then
-//                    assertEquals(actual = result, expected = albumList)
+                    assertTrue(result is UIResult.Success)
```

Comment:
Also check that the data is the same.
Since the assertions are form kotlinx test, it can do smart casting for you. 
So after the assert true kotlin should be able to know that the result is a success and should let you get the data out of it.

Reply:
implemented the required change